internal/fnruntime/utils.go

```diff
@@ -181,3 +198,25 @@ func kindMatch(node *yaml.RNode, selector kptfilev1.Selector) bool {
 func apiVersionMatch(node *yaml.RNode, selector kptfilev1.Selector) bool {
 	return selector.APIVersion == "" || selector.APIVersion == node.GetApiVersion()
 }
+
+// labelMatch returns true if the resource labels match input selection criteria
+func labelMatch(node *yaml.RNode, selector kptfilev1.Selector) bool {
+	nodeLabels := node.GetLabels()
+	for sk, sv := range selector.Labels {
+		if nv, ok := nodeLabels[sk]; !ok || sv != nv {
```

Comment:
ok --> found reflects the usage better (found label or not?)

Reply:
done